### PR TITLE
feat: make partner manager public

### PR DESCRIPTION
**Pull request by OnyekaIjeh**

### CODE REVIEW

File: contracts/Registrar/PartnerRegistrar.sol [hunk 1]
```diff
@@ -27,7 +27,7 @@ contract PartnerRegistrar is
 
     NodeOwner private _nodeOwner;
     IERC677 private _rif;
-    IPartnerManager private _partnerManager;
+    IPartnerManager public partnerManager;
```
Comment (sergio-jara) [inline contracts/Registrar/PartnerRegistrar.sol:30]: the only observation I would make is that the get partnermanager property becomes part of the API. We need to also define it in the INTERFACE. In order to ensure future implementation remain compatible.

**Review by sergio-jara — APPROVED** [review]

File: contracts/Registrar/PartnerRegistrar.sol (hunk 1 above)
Comment (sergio-jara) [reply contracts/Registrar/PartnerRegistrar.sol:30]: so... Maybe it becomes a function? getPartnerManager?

Comment (OnyekaIjeh) [reply contracts/Registrar/PartnerRegistrar.sol:30]: Yes, then the property can remain `private`. That's better.

Comment (OnyekaIjeh) [reply contracts/Registrar/PartnerRegistrar.sol:30]: We should expose the API only via the interface.

**Pull request merged**